README.md

```diff
@@ -10,7 +10,7 @@ The preparation lies in the group_var options (usernames/passwords, whether to i
 ### Usage:
 	$ git clone https://github.com/IQSS/dataverse-ansible.git dataverse
```

Comment:
Should we change `$` to `#` and mention above to run these commands as root?

Reply:
to me `$` means prompt; `#` means comment.

`-b` and `-K` were in there but may be expounded upon?